fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""Utility class for plotting metrics with error ranges."""
+
+from logging import error
+from typing import Any, Dict, List, Union
+from matplotlib.pyplot import subplot, title
+from numpy.lib.arraysetops import isin
+import pandas as pd
+import numpy as np
+
+from matplotlib.axes._axes import Axes
+
+from ._metric_frame import MetricFrame
+
+_MATPLOTLIB_IMPORT_ERROR_MESSAGE = "Please make sure to install matplotlib to use " \
+                                   "the plotting for MetricFrame."
```

Comment:
Might want to mention `pip install fairlearn[customplots]` (unless we've changed the option?)

Reply:
Not sure, but I copied this line from the `plot_threshold_optimizer`

Reply:
Being consistent across all such occurrences in the package makes sense I think (so +1 on copying). The text is technically correct and even `fairlearn[customplots]` only installs matplotlib.